### DISCUSSION TITLE
Filtering

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1091] [Office hours] [answered]
I'd like to discuss with you regarding 

1. the filter interactions on the updated debugger design. You can find it [here](https://twilio.invisionapp.com/share/BXZWLO6PM8W#/442028318_R6_-_Left_Nav_Testing_1)
2. Padding around the default Discloser component. You can find it [here](https://twilio.invisionapp.com/share/T8ZWLPNMP9W#/441823100_R6_-_Inspector_Request_1)
3. Combobox with option groups vs Select on the Time zone component. You can find it [here]([url](https://twilio.invisionapp.com/share/8AZWLRMB379#/441823088_R6_-_UI_Flows)

**Comment by SiTaggart** [comment, marked as answer]
Hi @sang-yeo, in order to be able to answer any of your questions or requests for feedback asynchronously we are going to need a little more information from you.

The more specific these discussion topics are, the more asynchronous answers we can give people, and the more of this information is searchable by future people who might come across similar situations. It's a super valuable resource we can build by doing this, and an important muscle to flex.

> 1. the filter interactions on the updated debugger design.

The invision page it links to doesn't seem to have any click targets for the filtering so I can't really tell what the interaction is. Regardless of that, what kind of feedback are you after here?

> 2. Padding around the default Discloser component.

Do you have a specific question or concern regarding the padding?

> 3. Combobox with option groups vs Select on the Time zone component.

What is the context around this decision?

**Reply by sang-yeo** [reply]
I'm sorry I must added wrong link. here is the correct one. https://twilio.invisionapp.com/share/67ZWNBV3EYH#/441823097_R6_-_Main BTW, this question is for Aayushi and Katie. They gave feedback on filtering during my ux review but I found out that they can't make it to the office hour so we can skip it. Although good to hear your feedback on filtering interaction. 

For padding, I wasn't clear on padding and margin rule when I use the default disclosure component. 1) we don't have sketch symbol for the default disclosure. 

Not sure I understand your question. My question is which component do I need to use for Time zone. I see two options, one is Combobox with option groups and the other one is Select with option groups

**Reply by kate-porter** [reply]
On the Disclosure piece, there is a Sketch library in Paste for the Disclosure. You'll need to link the library to your project in Abstract (follow the steps here: https://paste.twilio.design/getting-started/design#linking-the-libraries).

Within the Disclosure symbols, there is a grid you can turn on and off in the overrides panel to help you with spacing.

**Reply by kate-porter** [reply]
On the question of whether to use Select or Combobox for the Timezone field, take a look at the comparison between the two on the Select docs page, and let us know if you have any further questions: https://paste.twilio.design/components/select#whats-the-difference-between-select-and-combobox. For what it's worth, the Select component seems just fine based on what I'm seeing in your design.

**Reply by sang-yeo** [reply]
Yes, I did connect Sketch library in Paste for the Disclosure a few weeks ago. I didn't see the default disclosure symbol until today that's why I asked. Thanks for updating it.

**Comment by sang-yeo** [comment]
Looks like I need what I wanted from you all. Thank you for answering. I will cancel my slot and give it to Paul if it's not too late
